packages/py-api/py_api/functionality/hackathon/participants_base.py

```diff
@@ -35,7 +35,11 @@ def delete_participant(cls, object_id: str) -> results.DeleteResult | None:
     def update_participant(cls, object_id: str, participant: UpdateParticipant | NewParticipant) -> Any:
         obj = {
             key: value for key, value in participant.model_dump().items() if
-            value
+            value != None
```

Comment:
This is not needed. "If value" does the same as value != None, it's just more concise. In Python None, False, Empty List, Empty Dict, Empty Set and so on are falsy values.

Reply:
Damn, what is this dude talking about?

Reply:
A Микаел Again.

Reply:
> Hmmm. Are you sure about the logic here? We might want to replace a field with an empty list which in your case will be skipped. I think Isak's explicit check is the right way to go.

This is exactly why I hvae done it this way. Because that was the case if you want to set a boolean field back to false, it would ignore it.